backend/test/routes/zipDownload.js

```diff
@@ -82,10 +82,10 @@ describe('downloading a zip file', () => {
             expect(res.headers['content-disposition'])
               .to.eq('attachment; filename="myShow.zip"', 'file name must be show name')
           })
-          .then(res => JSZip.loadAsync(res.body))
+          .then(res => JSZip.loadAsync(res.body, {createFolders: true}))
```

Comment:
The tests are checking for the folders and the files, so the zip library needs to be told to create the folders rather than just files named folder/file.ext. It looks like this was a problem before we even messed with the zipfiles, its probably caused by an update in the zip library.